scripts/makebumpver

```diff
@@ -36,16 +36,116 @@ import sys
+
+        with open(self.configure, "rt") as f:
+            config_ac = f.read()
+
+        # get current package name, version & bug reporting email from configure.ac in case they are needed
+        regexp = re.compile(r"AC_INIT\(\[(.*)\], \[(.*)\], \[(.*)\]\)", flags=re.MULTILINE)
+        values = re.search(regexp, config_ac).groups()
+        current_name = values[0]
+        current_version = values[1]
+        current_bug_reporting_mail = values[2]
+        # also get the current release number
+        regexp = re.compile(r"PACKAGE_RELEASE, \[(.*)\]")
+        self.current_release = re.search(regexp, config_ac).groups()[0]
+
+        #argument parsing
+        parser = argparse.ArgumentParser(description="Increments version number and adds the RPM spec file changelog block. "
+                                                     "Also runs some checks such as ensuring rhel*-branch commits correctly "
+                                                     "reference RHEL bugs.",
+                                         epilog="The -i switch is intended for use with utility commits that we do not need to "
+                                                "reference in the spec file changelog.\n"
+                                                "The -m switch is used to map a Fedora BZ number to a RHEL BZ number for "
+                                                "the spec file changelog.\n"
+                                                "Use -m if you have a commit that needs to reference a RHEL bug and have cloned "
+                                                "the bug, but the original commit was already pushed to the central repo.")
+
+        parser.add_argument("-n", "--name", dest="name", default=current_name, metavar="PACKAGE NAME", help="Package name.")
+        #argument parsing
+        parser = argparse.ArgumentParser(description="Increment version number and add in RPM spec file changelog block. "
+                                                     "Also runs some checks such as ensuring rhel*-branch commits correctly "
+                                                     "reference RHEL bugs.",
+                                         epilog="The -i switch is intended for use with utility commits that we do not need to "
+                                                "reference in the spec file changelog.\n"
+                                                "The -m switch is used to map a Fedora BZ number to a RHEL BZ number for "
+                                                "the spec file changelog.\n"
+                                                "Use -m if you have a commit that needs to reference a RHEL bug and have cloned "
+                                                "the bug, but the original commit was already pushed to the central repo.")
+
+        parser.add_argument("-n", "--name", dest="name", default=current_name, metavar="PACKAGE NAME", help="Package name.")
```

Comment:
I think this is some copy & paste error. From what I see here the two commands above are the same as two commands above them with the same comment, or am I missing something?